server/activity_manager.go

```diff
@@ -79,7 +79,7 @@ func (m *ActivityManager) CreateActivity(ctx context.Context, create *api.Activi
 			return nil, fmt.Errorf("failed to find project ID %v for posting webhook event after changing the issue status %q", meta.issue.ProjectID, meta.issue.Name)
 		}
 		// TODO(dragonly): revisit the necessity of this function to depend on ActivityMeta.
-		projectRaw.CopyToProject(meta.issue.Project)
+		meta.issue.Project = projectRaw.ToProject()
```

Comment:
We may want to compose it here. We should compose it whenever we see ToProject().

Reply:
This is legacy code, which seems to only need an `IssueRaw` here.

Reply:
We may want to get rid of all XXRaw entirely in the server package. So we should make sure XX is composed whenever we use ToXX().

Reply:
Then we should refactor this function to only pass the necessary parameters, otherwise, it's a waste to compose only for the clean of code.

Reply:
Seems like a big refactor: need to change all the caller code to pass in a composed `Issue` (which itself is not refactored into `Issue` and `IssueRaw` yet).

I will create a PR for this later.

Reply:
Ok, we can keep it for now and fix it later when we have a composer layer.